### PR TITLE
IRO-1769 - All blocks page content

### PR DESCRIPTION
**Pull request by ndemidovich-sc**

### CODE REVIEW

File: pages/explorer/index.tsx [hunk 1]
```diff
@@ -1,9 +1,47 @@
...
+
+  return (
+    <>
+      <BlocksTable data={loaded ? data : data.concat(Array.from({ length: 20 }, () => null))} />
+      <span ref={observerRef} style={{ background: 'transparent', border: 'none', height: '0', width: '100%' }} />
+    </>
+  )
+}
 
 export default function Home() {
```
Comment (azhurauski-sc) [inline pages/explorer/index.tsx:30]: this name doesn't effect anything but lets call it explorer

File: pages/explorer/index.tsx [hunk 2]
```diff
@@ -1,9 +1,47 @@
...
+    loading: !loaded,
+    hasNextPage: metadata?.has_next,
+    disabled: !!error,
+    onLoadMore: loadNext,
+    rootMargin: "0px 0px 400px 0px"
+  });
+
+  return (
+    <>
+      <BlocksTable data={loaded ? data : data.concat(Array.from({ length: 20 }, () => null))} />
```
Comment (azhurauski-sc) [inline pages/explorer/index.tsx:24]: lets use constant for limit

File: pages/index.js [hunk 3]
```diff
@@ -189,9 +188,11 @@ export default function Home() {
               Blockchain statistics for $IRON
             </Text>
             <HStack>
-              <Button variant="secondary" size="medium">
-                View All Blocks
-              </Button>
+              <Link href="/explorer" passHref>
```
Comment (azhurauski-sc) [inline pages/index.js:191]: Please update links using enum added with Breadcrumbs, I added those when Breadcrumbs wasn't merged.
Also please check all block links in header and footer.

File: components/Breadcrumbs/BreadcrumbLink.tsx [hunk 4]
```diff
@@ -12,6 +13,16 @@ interface BaseBreadcrumbLinkProps {
...
 
+const LinkWrapper = ({ to, isCurrent, children }) => {
+  return isCurrent ? children : (
+    <NextLink href={to} passHref>
+      <Link>
+        {children}
+      </Link>
+    </NextLink>
+  )
+}
```
Comment (brekk) [inline components/Breadcrumbs/BreadcrumbLink.tsx:24]: I'd guess that this would be valuable as a standalone reusable component?

**Review by brekk — APPROVED** [review]

File: components/Breadcrumbs/BreadcrumbLink.tsx (hunk 4 above)
Comment (ndemidovich-sc) [reply components/Breadcrumbs/BreadcrumbLink.tsx:24]: I think yes

**Review by azhurauski-sc — APPROVED** [review]

**Review by brekk — APPROVED** [review]

**Pull request merged**